packages/magento-cart-items/CartItem/CartItemActionCard.tsx

```diff
@@ -84,10 +92,22 @@ export function CartItemActionCard(props: CartItemActionCardProps) {
             display: 'flex',
             alignItems: 'center',
             color: 'text.secondary',
+            mt: 1,
             gap: theme.spacings.xxs,
           })}
         >
-          <UpdateItemQuantity uid={uid} quantity={quantity} sx={{ my: 1 }} />
+          <UpdateItemQuantity
+            uid={uid}
+            quantity={quantity}
+            sx={(theme) => ({
+              [theme.breakpoints.down('sm')]: {
+                width: '70px',
```

Comment:
Double digit numbers don't fit when the width is 70px. Why is this necessary? Without this the row seems to fit fine.